src/lib/prepareOas.ts

```diff
@@ -149,8 +165,10 @@ export default async function prepareOas(path: string, command: 'openapi' | 'ope
     return capitalizeSpecType(type);
   });
 
+  const definitionVersion = await oas.version();
+
   // If we were supplied a Postman collection this will **always** convert it to OpenAPI 3.0.
-  const api = await oas.validate({ convertToLatest: false }).catch((err: Error) => {
+  const api = await oas.validate({ convertToLatest: opts.convertToLatest }).catch((err: Error) => {
```

Comment:
I'm going to leave this as-is because TS isn't picking up errors when options not available on `oas.validate()` are supplied here:

![Screen Shot 2022-12-08 at 2 33 34 PM](https://user-images.githubusercontent.com/33762/206581322-2fc78304-de09-4dc9-81bf-28de1bd18c11.png)

![Screen Shot 2022-12-08 at 2 32 48 PM](https://user-images.githubusercontent.com/33762/206581270-ec001abd-1a02-4d05-b43e-a9f979e47647.png)

![Screen Shot 2022-12-08 at 2 32 54 PM](https://user-images.githubusercontent.com/33762/206581280-3715ed2c-dc75-4af3-bcc9-ea9970a34c8c.png)